docs/data-structures/inorder_traversal.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: Inorder traversal of Binary Tree
+slug: /data-structures/inorder_traversal
```

Comment:
Uff. Do you know how slugs work? You don't insert underscores here

Reply:
Sir rajiv sir commrnt me to do that